### PR TITLE
Add HE transforms for capability, itsystem, orgunit, project, person

### PR DESCRIPTION
**Pull request by alightwing**
* I have tried to match the ontology as closely as possible, including stub values where no values exist in the source sheets.
* This pulls its data from five different workbooks, a situation that is only going to get worse as more classes are added. The relevant workbook is noted at the top of each transform. 
* The command to build the triples is extremely fun:

```python3 -m sheet_to_triples --book '../books/highways_SMP_ingest_20210310.xlsx' --book '../books/20210310 activity2project v2.xlsx' --book '../books/20210305 org_unit2activity - master.xlsx' --book '../books/20210311 activity2itsystem v3.xlsx' --book '../books/capability2activity.xlsx' --add-graph ../turtle.owl --model ../he_empty.json hwx_labels hwx_org_labels he_class_icons hwx_views he/he_orgunit he/he_orgunit2activity he/he_project he/he_project2activity he/he_itsystem he/he_itsystem2activity he/he_capability he/he_capability2activity he/he_person --verbose```

* This is adding enough transforms that I think it warrants its own folder, if only to separate it out from earlier `he` transforms.

### CODE REVIEW

**Review by bz2 — APPROVED** [review]
Thanks!

There should be some more transform-location structure really, yes. The one upside here is you *can* do it incrementally, making a partial `--model-out` and putting it back in to `--model` on the next run. So, don't _need_ one giant command.

**Comment by alightwing** [comment]
This should now work, although you'll need the minor hacks to get this working better with sheets. You'll also need an additional workbook for `person`. The rest of it should be fine.

**Comment by alightwing** [comment]
(PS I can't merge this until you give me write access :P)

**Comment by bz2** [comment]
@alightwing You now have write access, also have run this locally with a few make-sheet happy tweaks elsewhere and made data, so looking reasonable.

**Pull request merged**
